piqe_ocp_lib/api/resources/ocp_cluster_versions.py

```diff
@@ -170,12 +170,19 @@ def wait_until_upgraded(timeout: int):
             :return: CVO object OR None
             """
             ends_at = datetime.now() + timedelta(minutes=timeout)
-            health_checker = OcpHealthChecker()
+            health_checker = OcpHealthChecker(self.kube_config_file)
+
+            cluster_version = self.get_cluster_version()
+            for condition in cluster_version["status"]["conditions"]:
+                if condition["type"] == "Progressing" and condition["status"] == "True":
+                    break
+                else:
+                    continue
```

Comment:
nit-picky: you can remove the `else` here

Reply:
Done, thanks !!